lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -683,7 +672,7 @@ export class UserAgentApplication {
 
       // What is this? Is this the reject that is passed in?? -- REDO this in the subsequent refactor, passing reject is confusing
       if (reject) {
-        reject(ErrorCodes.endpointResolutionError + ":" + ErrorDescription.endpointResolutionError);
+        reject(ClientAuthError.createEndpointResolutionError());
```

Comment:
double check throw vs. reject.
also should we be doing return reject();

Reply:
We will leave it as reject for now, we should change our guidance to consumers to use .catch for the error cases rather than the reject function pointer.